package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "cro-nodelib",
+  "version": "0.1.0",
+  "description": "Crypto.com Chain library for Node.js",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crypto-com/cro-nodelib"
+  },
+  "author": "Calvin Lau <calvin@crypto.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "ow": "^0.15.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.5",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.12.8",
+    "@types/sinon": "^7.5.0",
+    "@typescript-eslint/eslint-plugin": "^2.8.0",
+    "@typescript-eslint/parser": "^2.8.0",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "eslint": "^6.1.0",
+    "eslint-config-airbnb-base": "^14.0.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "mocha": "^6.2.2",
+    "neon-cli": "^0.3.3",
+    "nodemon": "^1.19.4",
+    "prettier": "^1.19.1",
+    "sinon": "^7.5.0",
+    "ts-node": "^8.5.2",
+    "tslint": "^5.20.1",
+    "typescript": "^3.7.2"
+  },
+  "scripts": {
+    "build": "neon build",
+    "nodemon": "nodemon",
+    "eslint": "eslint 'lib/**'",
+    "test": "mocha --require ./node_modules/ts-node/register --exit --color --recursive 'lib/src/**/*.spec.ts'"
```

Comment:
I can't `yarn install` & `yarn test` successfully on node 13.2.0, yarn 1.19.1
what's the compatible node version? what did I miss?